netaddr.go

```diff
@@ -973,13 +973,20 @@ func MustParseIPPort(s string) IPPort {
 // IsZero reports whether p is its zero value.
 func (p IPPort) IsZero() bool { return p == IPPort{} }
 
+// Valid reports whether p.IP() is non-zero.
```

Comment:
Worth explicitly mentioning that the port is ignored, or that a zero port may be valid?

Reply:
Done.